arch/eurohpc/lumi/cray-gpu/15.0.1/toolchain.cmake

```
@@ -0,0 +1,43 @@
+# (C) Copyright 1988- ECMWF.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+####################################################################
+# COMPILER
+####################################################################
+
+set( ECBUILD_FIND_MPI OFF )
+set( ENABLE_USE_STMT_FUNC ON CACHE STRING "" )
+
+####################################################################
+# OpenMP FLAGS
+####################################################################
+
+set( ENABLE_OMP ON CACHE STRING "" )
+set( OpenMP_C_FLAGS   "-homp" CACHE STRING "" )
+set( OpenMP_Fortran_FLAGS   "-homp -hnoacc -hlist=aimd -maxrregcount 64" CACHE STRING "" )
+
```

We should add here
```suggestion
set( OpenMP_C_LIB_NAMES       "craymp" )
set( OpenMP_CXX_LIB_NAMES     "craymp" )
set( OpenMP_Fortran_LIB_NAMES "craymp" )
set( OpenMP_craymp_LIBRARY    "/opt/cray/pe/cce/15.0.1/cce/x86_64/lib/libcraymp.so" )

```
to avoid having to specify this on the build command.